pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -2061,7 +2061,6 @@ func (a *ApplicationBackupController) backupResources(
 		if err != nil {
 			return err
 		}
-		return nil
```

Comment:
Couple of questions:
1. we are doing the same in case of NFS scenario too  Do we need to  take care  of it there too ? 
2. Now in case of large resource we will go ahead with rancher changes will it have any impact in rancher cluster with large resource combination. since we removed the return statement.

Reply:
As per our discussion, for NFS we will have two times getResource() invocation.

one during large resource determination time and other is  actual resource upload time via resource backup(& RE) CR. this can be irradiated only by propagating back the large resource flag from RB --> RE --> AB and not executing this logic for NFS case in AB CR handler function.

this we can do as part of later releases.

2. regrading rancher please ignore it seems we will call it nevertheless in second time reconciler enters.

Reply:
Yes, for NFS, We need to modify the RE/RB CR to get the largeResources details from executor and update the ApplicationBackup CR. Will pick it up in the next release.